public/js/controllers/dashboardEditorController.js

```diff
@@ -748,10 +750,15 @@ angular.module('madisonApp.controllers')
             $scope._setFeaturedDoc();
           }
         });
+        */
       };
 
       $scope._setFeaturedDoc = function () {
         return $http.post('/api/docs/featured', {id: $scope.doc.id});
       };
+
+      $scope.removeFeaturedDoc = function() {
+        return $http.delete('/api/docs/featured/' + $scope.doc.id);
```

Comment:
When this is run by the "Remove" button in the doc edit settings screen, it successfully hits the API route but doesn't update the state on the page to show it was successfully removed.